lua/wikis/commons/Infobox/Map.lua

```diff
@@ -75,29 +79,49 @@ end
 
 --- Allows for overriding this functionality
 ---Add wikispecific categories
+---@protected
 ---@param args table
----@return table
+---@return string[]
 function Map:getWikiCategories(args)
 	return {}
 end
 
 --- Allows for overriding this functionality
 ---Adjust Lpdb data
+---@protected
 ---@param lpdbData table
 ---@param args table
 ---@return table
 function Map:addToLpdb(lpdbData, args)
 	return lpdbData
 end
 
+--- Allows for overriding this functionality
+---Returns the respective game for this map
+---@param args table
+---@return string?
+function Map:getGame(args)
+	return Game.name{game = args.game}
+end
+
+--- Allows for overriding this functionality
+---Returns the respective game mode(s) for this map
+---@param args table
+---@return string[]
+function Map:getGameModes(args)
+	return self:getAllArgsForBase(args, 'mode')
+end
+
+---@private
 function Map:_readCreators()
 	self.creators = {}
 	for _, creator in Table.iter.pairsByPrefix(self.args, {'creator', 'created-by'}, {requireIndex = false}) do
-		table.insert(self.creators, creator)
+		table.insert(self.creators, Page.pageifyLink(creator))
```

Comment:
this makes it display with underscores
ideally we would do something like
```suggestion
		table.insert(self.creators, {page = Page.pageifyLink(creator), displayName = creator})
```
and in the display just use
`Array.map(self.creators, Operator.property('displayName'))`

and for the storage something like
```
		extradata = Table.merge(
			Table.map(self.creators, function(index, value)
				return 'creator' .. index, value.page
			end),
			Table.map(self.creators, function(index, value)
				return 'creator' .. index .. dn, value.displayName
			end),
			{
				game = self:getGame(args),
				modes = self:getGameModes(args),
			}
		)
```